kahuna/public/js/upload/jobs/required-metadata-editor.js

```diff
@@ -30,7 +31,7 @@ jobs.controller('RequiredMetadataEditorCtrl',
         var cleanMetadata = {};
         Object.keys(ctrl.metadata).forEach(key => {
             if (ctrl.metadata[key] !== ctrl.saveWhenChangedFrom[key]) {
-                cleanMetadata[key] = ctrl.metadata[key];
+                cleanMetadata[key] = ctrl.metadata[key] || '';
```

Comment:
Not if the `input` is `required`. If `required`, Angular uses `undefined` as the empty value. I swear we'd fixed this before but evidently not.